Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+Hi there! This guide aims to teach people unfamiliar with Github how to suggest edits to the [Tanzu Agile Practices site](https://tanzu.vmware.com/developer/practices/). Follow the steps below to get started!
```

Comment:
```suggestion
Hi there! This guide aims to teach people unfamiliar with GitHub or the Git command line how to suggest edits and submit Pull Requests using only the GitHub Web UI. We will use the [Tanzu Agile Practices section](https://tanzu.vmware.com/developer/practices/) for our examples. Follow the steps below to get started!
```

Reply:
While the original context is for the Practices content, this tutorial is generally applicable to any text-based TDC content.